Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -2974,36 +2987,47 @@ def __init__(self, ip, dbot_score, asn=None, as_owner=None, region=None, port=No
                      hostname=None, geo_latitude=None, geo_longitude=None,
                      geo_country=None, geo_description=None, detection_engines=None, positive_engines=None,
                      organization_name=None, organization_type=None, feed_related_indicators=None, tags=None,
-                     malware_family=None, relationships=None):
+                     malware_family=None, relationships=None, blocked=None, description=None, stix_id=None,
+                     whois_records=None):
+
             self.ip = ip
+
+            self.blocked = blocked
```

Comment:
Is there any meaning for the grouping here?
If so, Add a comment before each one.